spec/mailers/user_mailer_spec.rb

```diff
@@ -19,7 +19,7 @@
     it "renders the headers" do
       expect(mail.subject).to eq('Shop with Supermarkit')
       expect(mail.to).to eq(["#{user.email}"])
-      expect(mail.from).to eq(["us@supermarkit.ca"])
+      expect(mail.from).to eq(["folks@supermarkit.ca"])
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.